source/retryable-reads/tests/changeStreams-client.watch-serverErrors.yml

```diff
@@ -147,3 +147,33 @@ tests:
         operations: [*retryable_operation_fails]
         expectations:
              - *retryable_command_started_event
+   -
+        description: "client.watch fails after ExceededTimeLimit"
+        clientOptions:
+            retryReads: false
+        failPoint:
+            <<: *failCommand_failPoint
+            data: { failCommands: [aggregate], errorCode: 262 }
+        operations: [*retryable_operation_fails]
+        expectations:
+             - *retryable_command_started_event
+   -
+        description: "client.watch fails after LockTimeout"
+        clientOptions:
+            retryReads: false
+        failPoint:
+            <<: *failCommand_failPoint
+            data: { failCommands: [aggregate], errorCode: 24 }
+        operations: [*retryable_operation_fails]
+        expectations:
+             - *retryable_command_started_event
+   -
+        description: "client.watch fails after ClientDisconnect"
+        clientOptions:
+            retryReads: false
```

Comment:
Per https://github.com/mongodb/specifications/pull/595/files#r304042576, let's remove `retryReads: false` from these three tests and make the expectations consistent with `db.watch`.

Reply:
done.